scripts/src/core/utils.ts

```diff
@@ -70,7 +70,7 @@ module fugazi.utils {
 	
 		reset(): void {
 			this.cancel();
-			this.name = setTimeout(this.callback, this.interval);
+			this.name = setTimeout(this.callback, this.interval) as any as number;
```

Comment:
the nodejs typings says that setTimeout returns `Timer`, which is not a number.
This is a fast workaround